src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -13829,6 +13832,7 @@ public static void M(ref S1 s1)
                 // (15,23): error CS8352: Cannot use variable 's2' in this context because it may expose referenced variables outside of their declaration scope
                 //         M2(ref s1, $"{s2}");
                 Diagnostic(ErrorCode.ERR_EscapeVariable, "s2").WithArguments("s2").WithLocation(15, 23)
+ 
```

Comment:
nit: unnecessary blank line

Reply:
And the correctness test decided to escalate your nit to a blocking error 😦